src/extension.ts

```diff
@@ -46,7 +46,15 @@ const findPubspec = async (activeFileUri: vscode.Uri) => {
 }
 
 export const relativize = (filePath : String, importPath : String) => {
-    const pathSplit = (path : String) => path.length === 0 ? [] : path.split('/');
+    const pathSplit = (_path : String) => {
+        if (_path.length === 0) return [];
+        const slash = _path.split('/');
+
+        // Handles OS specific paths (Windows related)
+        const osSeparator = _path.split(path.sep);
+        if (slash.length > osSeparator.length) return slash;
+        else return osSeparator;
```

Comment:
`_path.split(path.sep)` should work on all OS's, right? you don't need the if and line 51.
I can test on linux & osx for you after you update the pr :)

Reply:
Well, clearly idk how this works, haha, but I replied this in the pull request